pyvo/registry/__init__.py

```diff
@@ -15,4 +15,5 @@
            "Freetext", "Author",
            "Servicetype", "Waveband", "Datamodel", "Ivoid", "UCD",
            "Spatial", "Spectral", "Temporal",
-           "choose_RegTAP_service", "RegTAPFeatureMissing"]
+           "choose_RegTAP_service", "RegTAPFeatureMissing",
+           "RegistryResults", "RegistryResource",]
```

Comment:
These need to be imported, too. 

And that should fix the traceback in the docs build.